### PR TITLE
Mikael changing header title (test)

### PR DESCRIPTION
**Pull request by Maatala**
Whatever

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -12,7 +12,7 @@ <h1></h1>
     </header>
 
     <main class="header__page-title">
-        Hello
+        This is Mikael speaking!
```
Comment (Alexander-Lindstroem) [inline index.html:15]: LGTM

File: project.html [hunk 2]
```diff
@@ -12,7 +12,7 @@ <h1></h1>
     </header>
 
     <main class="header__page-title">
-        Hello
+        This is Mikael speaking!
```
Comment (Alexander-Lindstroem) [inline project.html:15]: LGTM

**Review by Alexander-Lindstroem — COMMENTED** [review]
Good job

File: index.html (hunk 1 above)
Comment (Alexander-Lindstroem) [inline index.html:15]: Actually I dont like this on this page, please revert

Comment (Maatala) [reply index.html]: Alright, I fixed it

**Pull request merged**